tests_oval_graph/test_commands/test_command_line_client/test_client_json_to_html.py

```diff
@@ -0,0 +1,75 @@
+import tempfile
+import uuid
+from pathlib import Path
+
+import pytest
+
+from oval_graph.command_line_client.json_to_html import JsonToHtml
+
+from ...test_tools import TestTools
+from .global_test_feed import EXPECTED_RULES_ID, SEARCH_RULES
+
+PATH_TO_REPORT = Path('../test_data/referenc_result_data_json.json')
+TOP_PATH = Path(__file__).parent
+
+
+def get_client_json_to_html(rule, optional_args=None, src=PATH_TO_REPORT):
+    patch = str(TOP_PATH / src)
+    args = ["--display", patch, rule]
+    if optional_args is not None:
+        args.extend(optional_args)
+    client = JsonToHtml(args)
+    client.load_file()
+    return client
+
+
+@pytest.mark.parametrize("src, error_pattern", [
+    ('../../global_test_data/ssg-fedora-ds-arf.xml', 'is not valid json'),
+    ('../test_data/referenc_html_report.html', 'No such file or directory:'),
+    ('../test_data/bad_result_data_json.json', 'valid for OVAL tree')
```

Comment:
The paths to these files repeat multiple times and in multiple files across this PR. Could we define them on a single place? Would there be a way to avoid the relative paths (the two dots `..`)?